src/components/Header/LongHeader.jsx

```diff
@@ -0,0 +1,134 @@
+        <TopContainer>
+          <StyledMenu />
+          <h1>Comepet</h1>
+          {isLoggedIn ? (
+            <IconContainer>
+              <StyledInformation />
+              <StyledNotification />
+            </IconContainer>
+          ) : (
+            <Button
+              bgColor="#fff"
+              type="button"
+              width="2.8rem"
+              height="2.8rem"
+              borderRadius="50%"
+              margin="0 1rem 0 0">
+              <Image alt="로그인" width="2.8rem" height="2.8rem" type="profile" />
+            </Button>
+          )}
+        </TopContainer>
+        <MiddleContainer>
+          <BackgroundBox width="15rem" boxShadow="0px 4px 16px rgba(0, 0, 0, 0.08)">
+            <Button
+              bgColor="#fff"
+              type="button"
+              color="#2A2E56"
+              fontWeight="bold"
+              borderRadius="1.6rem">
+              <Image
+                src={getImageSrc('/images/finding.png')}
+                width="1.6rem"
+                height="1.6rem"
+                margin="0 5rem 0 0"
+              />
+              실종 및 보호
+            </Button>
+          </BackgroundBox>
+          <BackgroundBox width="15rem" boxShadow="0px 4px 16px rgba(0, 0, 0, 0.08)">
+            <Button
+              bgColor="#fff"
+              type="button"
+              color="#2A2E56"
+              fontWeight="bold"
+              borderRadius="1.6rem">
+              <Image src={getImageSrc('/images/home.png')} width="1.6rem" height="1.6rem" />
+              보호소 동물
+            </Button>
+          </BackgroundBox>
+        </MiddleContainer>
+        <BottomContainer>
+          <Input placeholder="세부 검색을 위해 클릭해주세요" borderRadius="1.6rem" />
+          <StyledSearch />
+        </BottomContainer>
+      </BackgroundBox>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div`
+  position: fixed;
+  width: 100%;
+  top: 0;
+`;
+
+const TopContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const MiddleContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-top: 0.8rem;
+`;
+
+const BottomContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: 1.6rem 2.8rem 0 2.8rem;
+`;
+
+const StyledMenu = styled(MenuIcon)`
+  font-size: 3rem;
+  margin-left: 1rem;
+  color: ${({ theme }) => theme.colors.brand};
+`;
+
+const StyledSearch = styled(SearchIcon)`
+  position: absolute;
+  right: 0;
+  margin: 0.4rem 3.6rem 0 0;
+  font-size: 3rem;
+  color: ${({ theme }) => theme.colors.brand};
+`;
+
+const IconContainer = styled.div`
+  margin-right: 1.2rem;
+`;
+
+const StyledNotification = styled(NotificationsActiveIcon)`
+  font-size: 2.8rem;
+  color: ${({ theme }) => theme.colors.brand};
+`;
+
+const StyledInformation = styled(ErrorOutlineIcon)`
+  font-size: 2.8rem;
+  margin-right: 0.8rem;
+  color: ${({ theme }) => theme.colors.brand};
+`;
+
```

Comment:
```suggestion
const ButtonText = styled.div`
  margin: 0.3rem 0 0 0.5rem;
`;
```
Button 안에서 Image는 수직정렬 됐지만 텍스트는 수직정렬 되지 않아 생긴 문제로 Image에 margin을 주지 않고 div에 margin을 줬습니다. 

적용 결과입니다.
![image](https://user-images.githubusercontent.com/81365896/144699917-cc7c20e9-3236-47a7-aba0-24e0f6e44345.png)

Reply:
Input의 Icon 수직 정렬에 대한 해결법입니다.

```suggestion
const InputWrapper = styled.div`
  position: relative;
  width: 100%;
`;
```

Reply:
Button 에 display flex를 통해서 수직 정렬 된 것으로 보이는데, 혹시 수직 정렬이 안돼 보였을까요?

지금 확인하는데 위에 제시해주신 코드 없이도 수직 정렬이 잘 된 것으로 보여서요!

![image](https://user-images.githubusercontent.com/81841082/144708964-1dc850f7-d741-40da-8629-4b2893420ece.png)

Reply:
f08f3db 기준으로 이렇게 보이네요

https://user-images.githubusercontent.com/81365896/144725975-907a7599-b23a-4d9c-a33b-bb46decd3300.mov